mcpscanner/core/static_analysis/native_analyzer.py

```
@@ -3896,6 +3922,262 @@ def _ts_extract_parameters(self, node: "Node") -> List[Dict[str, Any]]:
+        if not reachable:
+            return
+
+        dangerous_leaves: Set[str] = set()
+        reachable_names = list(ctx.reachable_functions or [])
+        for leaf, target in reachable:
+            if leaf not in reachable_names:
+                reachable_names.append(leaf)
+            try:
+                ops = self._ts_detect_security_ops(target)
+            except Exception:  # pragma: no cover - defensive
+                continue
```

Ruff (0.15.18)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)